src/model/nonce.ts

```diff
@@ -1,18 +1,23 @@
 import * as redis from "redis";
 import * as TE from "fp-ts/TaskEither";
 import { toError } from "fp-ts/lib/Either";
-import { pipe } from "fp-ts/lib/function";
+import { pipe, flow } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as E from "fp-ts/lib/Either";
+import { readableReportSimplified } from "@pagopa/ts-commons/lib/reporters";
 import {
   FUNCTION_PREFIX,
   falsyResponseToErrorAsync,
   singleStringReply
 } from "../utils/redis/client";
 import { Nonce } from "../generated/definitions/models/Nonce";
+import { LollipopSignatureInput } from "../generated/definitions/internal/LollipopSignatureInput";
 
+const nonceRegex = new RegExp('nonce="(.*?)"');
```

Comment:
ditto

Reply:
moved in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/f31eed4e20119c91d4e2b9977660f7823ed56d1a